net/linuxptp/Makefile

```diff
@@ -38,15 +38,36 @@ define Package/linuxptp/description
  computers.
 endef
 
+define Package/linuxptp/conffiles
+/etc/config/linuxptp
+endef
+
 EXTRA_CFLAGS += -DHAVE_CLOCK_ADJTIME -DHAVE_POSIX_SPAWN -DHAVE_ONESTEP_SYNC
 
 MAKE_VARS += \
 	EXTRA_CFLAGS="$(TARGET_CFLAGS) $(TARGET_CPPFLAGS) $(EXTRA_CFLAGS)" \
 	KBUILD_OUTPUT="$(LINUX_DIR)" \
 	NO_AUTODETECT_SAD_MAC_LIB="y"
 
+# separate compile step for helper utility
+define Build/Compile/sysclock-discipline-check
+	$(TARGET_CC) $(TARGET_CFLAGS) -o $(PKG_BUILD_DIR)/sysclock-discipline-check $(CURDIR)/files/sysclock-discipline-check.c $(TARGET_LDFLAGS)
+endef
```

Comment:
I am not sure, if I like this way. You added it here, then added one more file to this repository instead of sending it to the upstream and then backporting it here as patch.